M/msgpack/build_tarballs.jl

```diff
@@ -0,0 +1,40 @@
+using Pkg
+using BinaryBuilder
+
+const NAME = "msgpack"
+const GIT = "https://github.com/msgpack/msgpack-c.git"
+const GIT_TAGS = Dict(
+    "cpp-3.0.1" => "2c4f2b890ef1546fc022d270d11e657f6fc8022f",
+)
+
+const PLATFORMS = [
+    Platform("x86_64", "linux"; libc="glibc", cxxstring_abi="cxx11"),
+    Platform("x86_64", "linux"; libc="musl", cxxstring_abi="cxx11"),
+]
```

Comment:
Why only these two platforms?

Reply:
These are the platforms that we target for ROCm, but we can build for all supported platforms.

Reply:
If this is potentially useful beyond ROCm it may be worth to try and build for more platforms.